build.gradle

```diff
@@ -111,8 +100,14 @@ configurations.all {
     }
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.17
+targetCompatibility = 1.17
+
+run {
+    jvmArgs = [
+            '--add-opens', 'java.base/java.io=ALL-UNNAMED'
```

Comment:
What in picard is requiring this?   Is there any way we can avoid it?  I feel like this requirement is going to cause a ton of confusion for picard users.

Reply:
MergeBamAlignmentTest - I've changed it now so its only for test tasks.